src/lib/controllers/ui-controller.js

```diff
@@ -1,38 +1,48 @@
-/* global Node, Event */
+/* global Event */
 import { Lexeme, Feature, Definition, LanguageModelFactory, Constants } from 'alpheios-data-models'
+import { AlpheiosTuftsAdapter } from 'alpheios-morph-client'
+import { Lexicons } from 'alpheios-lexicon-client'
+import { Grammars } from 'alpheios-res-client'
```

Comment:
Agree that a UI controller should be modular. Still, this is not a full UI controller refactoring, that's a more local change of a content refactoring and a UI controller change so that we'll be able to split controller creation and resource loading: constructor creates a controller and it is fast and allocates almost no resources and `init()` loads all resources necessary and is much slower.